.github/workflows/ci-storybook.yml

```diff
@@ -0,0 +1,38 @@
+name: Build storybooks
+
+on:
+  pull_request
+
+jobs:
+  build_storybooks:
+   runs-on: ubuntu-latest
+   env:
+     APP_ENV: production
+     NODE_ENV: production
```

Comment:
From memory, this is set so that `yarn install` runs in the same environment as the builds but I see that the install uses `--production=false` anyway. I think you're right that it can be deleted.